test/TESTING_GUIDELINES.md

```diff
@@ -1,85 +1,33 @@
+1. Google Chrome (latest version)
+2. Chromedriver (WebDriver for Chrome)
+  - `brew cask install chromedriver`
+  - **Note:** if you have chromedriver already installed, make sure it's the latest version: `brew cask upgrade chromedriver`
 
-**Custom run:**
+### Running tests locally
 
-In `test/` directory:
-1. Run `bundle install` to install ruby dependencies
-2. Run `bundle exec cucumber BROWSER=chrome SDK_URL=${SDK_URL} USE_SECRETS=false SEED_PATH=false DEBUG=false`, where you can customize:
-    - `BROWSER` - browser to run tests against (possible choices: `chrome`, `firefox`)
-    - `SDK_URL` - url to deployed SDK (possible choices: any surge link, or `"https://localhost:8080/?async=false"` when webpack started locally using `npm run dev`.
-    **Note:** It's very important to add `?async=false` GET parameter to the end of any URL so that tests wait for JWT to be generated)
+ 1. Run `test.sh` script OR
+ 2. Run: 
+  - `npm run build:dev`
+  - `npm run test:ui`
 
-## Maintenance
-
-When developing a new feature, in most cases you will perform changes only in places mentioned under [Test structure](#test-structure) section. If new flow is introduced, for example cross-device flow, a separate page object, step definition and `.feature` file can be created to handle the testing.
-
-If there's a need to introduce a change in `monster` gem, follow the steps:
-- create a branch out of master on `monster` gem
-- bump version in `monster/lib/monster/version.rb` according to your changes
-- for development purposes change `monster` gem dependency line in `test/Gemfile` of this project:
-  - if changes on monster branch: `gem 'monster', :git => 'git@bitbucket.org:onfido/monster.git', :branch => 'your-branch-on-monster'`
-  - if changes on local changes: `gem 'monster', :path => 'your/local/path/to/monster'`
-- run `bundle update monster` in `test/` directory
-- once solution tested, issue PR with your monster branch to `master` and assign 2 QAs
-- if PR approved, create tag with version from `version.rb`
-  - `git tag <version>` version
-- merge your `monster` branch to `master`
-- back in this project, update `monster` gem dependency line in `test/Gemfile` to reflect latest changes in monster by tag:
-  -  `gem 'monster', :git => 'git@bitbucket.org:onfido/monster.git', :tag => '<version_from_your_branch>'`
-- run `bundle update monster` in `test/` directory to apply the changes (`Gemfile.lock` should be updated)
-
-### Test structure
-
-All test artifacts are in `features/` directory:
-- `page_objects/` - contains files with page objects that map objects i.e by id or css
-- `step_definitions/` - contains generic, commonly shared steps used in `.feature` files
-- `*.feature` - behavior-driven tests written in Gherkin
-- `support/` - contains support for setting up tests, i.e requires `monster` gem dependency
-- `scripts/` - contains scripts to setup test environment on Travis CI
+### Test structure in `test` directory
```

Comment:
Minor: "`test` directory structure" describes this section better IMO

Reply:
👍 done 0012fa2